infra/modules/gcp-oauth-long-access-strategy/README.md

```diff
@@ -1,4 +0,0 @@
-# gcp-oauth-long-access-strategy
-
-Infra to support `OAuthAccessTokenSourceAuthStrategy` for a gcp psoxy instance.
```

Comment:
This module and the other one for refresh token weren't used and with the new one, not valid anymore.